api/src/identity-access-management/domain/errors.js

```diff
@@ -39,8 +39,8 @@ class OrganizationLearnerIdentityNotFoundError extends DomainError {
 }
 
 class MissingOrInvalidCredentialsError extends DomainError {
-  constructor(message = 'Missing or invalid credentials') {
-    super(message);
+  constructor(message = 'Missing or invalid credentials', code = 'MISSING_OR_INVALID_CREDENTIALS') {
+    super(message, code);
```

Comment:
cf. https://github.com/1024pix/pix/blob/dev/docs/errors.md

```suggestion
  constructor(message = 'Missing or invalid credentials', code = 'MISSING_OR_INVALID_CREDENTIALS') {
    super(message, code);
```

Reply:
Je viens de mettre à jour ma suggestion pour le `constructor` de `MissingOrInvalidCredentialsError`. J'avais fait une erreur de copier-coller dans mon premier jet. Désolé.